.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,6 +9,8 @@
 
 ### Changes proposed in this Pull Request:
 
+<!-- If necessary, indicate if this PR is part of a bigger feature. Add a label with the format `focus: name of the feature [team:name of the team]`. -->
```

Comment:
Nitpick: I think it would be clearer to have the instruction in the Submission Review Guidelines section. For example:

`I have added a label with the format `focus: name of the feature [team:name of the team]`

Reply:
Also, I realized the Submission Review Guidelines section contains links to guides. In this case, it's not a guide but a single instruction formatted as a comment. I understand the value of placing it at the top of the document to highlight its importance but I can't see why it would fit that section. Thanks again for this suggestion @adrianduffell! It made me think carefully about this.

Reply:
Thanks @alopezari , your reasoning is solid!